syntax = "proto3";
package jio.brain.proto.knowledge.healthcare;

import "jio/brain/proto/knowledge/healthcare/base/attribute.proto";
import "jio/brain/proto/knowledge/healthcare/base/status.proto";

message GetAttributeNameRequest {
    uint32 symptom_type_id = 1;
    uint32 attribute_type_id = 2;
}

message GetAttributeNameResponse {
	string attribute_name = 1;
    jio.brain.proto.knowledge.healthcare.base.Status status = 2;
}